eq-author/cypress/integration/authenticated/errors_spec.js

```diff
@@ -0,0 +1,23 @@
+import { testId } from "../../utils";
+describe("errors", () => {
+  beforeEach(() => {
+    cy.visit("/");
+    cy.login();
+  });
+
+  it("displays page not found page if navigating to non-existant questionnaire", done => {
+    cy.on("uncaught:exception", () => {
+      // return false to prevent the error from
+      // failing this test
+      return false;
+    });
+
+    cy.visit("http://localhost:3000/#/q/not-existant-id");
```

Comment:
Ah this domain driven by an environment variable which is defined here https://github.com/ONSdigital/eq-author-app/blob/e85f6f9d2ef9b9e86e59bcebee4722aabeffd7f2/eq-author/.env.test#L1

However you would probably want to read it with https://docs.cypress.io/guides/references/configuration.html#Cypress-config

Reply:
Now changed to relative URL.